### ISSUE TITLE
No Popup display to rate in app download from playstore

### ISSUE THREAD

**Issue by rb162215** [MinaSamir11/react-native-in-app-review#30]
In my android app after setting up code as per doc when i upload app to playstore and then after app open no popup display.

below is my code - ` if(InAppReview.isAvailable()){
            console.log('InAppReview',InAppReview.isAvailable());
            InAppReview.RequestInAppReview();
}`

and while connective usb to device i can see log- 

` I/ReactNativeJS: 'InAppReview', true`
`ReviewService : requestInAppReview (com.mypackagename)
Initiate binding to the service.
isGooglePlayServicesAvailable: true`

Any suggestion, where i am wrong.

**Comment by MinaSamir11** [comment]
* Duplicated issue, please read this comment to how you can test Popup since it is not display in debugging mode. 
 - https://github.com/MinaSamir11/react-native-in-app-review/issues/6#issuecomment-699482016
 - https://github.com/MinaSamir11/react-native-in-app-review/issues/4#issuecomment-683164631

**Comment by MinaSamir11** [comment]
try this sol. 
- https://github.com/MinaSamir11/react-native-in-app-review/issues/25#issuecomment-760077474

**Comment by rb162215** [comment]
> try this sol.
> 
> * [#25 (comment)](https://github.com/MinaSamir11/react-native-in-app-review/issues/25#issuecomment-760077474)
> 
> and try with different Gmail account that has not reviewed yet your app.

My app is on playstore and tried with multiple devices, in mi device its display only once but after pressing not now button, its not display second time. in other device(samsung, moto) is is not displaying.

**Comment by rb162215** [comment]
Is there any support for this, even after 10days of installation from playstore no popup to rate in android.
In ios app popup display 3 times but after pressing not now button, no popup display again.

is there any documentation how and when popup will display in different scenario.
Still facing same problem.

**Comment by MinaSamir11** [comment]
In README, you will find guildlines and rules when to show Review flow to user, if you break rules, your app may be blocked in app review feature to that user, who receives inappreview flow every time, please read doc well. and open img will redirect to android and ios offical doc

**Comment by rb162215** [comment]
Still not display dialog in android app. Checked in multiple devices but not a single time rate view display.
below is log (App downloaded from playstore).

` I/PlayCore: UID: [10166]  PID: [5472] ReviewService : requestInAppReview (com.companyname.myapp)
 I/PlayCore: UID: [10166]  PID: [5472] ReviewService : Initiate binding to the service.
 E/isGooglePlayServicesAvailable: true
 D/CompatibilityInfo: mCompatibilityFlags - 0
 D/CompatibilityInfo: applicationDensity - 320
 D/CompatibilityInfo: applicationScale - 1.0
 I/PlayCore: UID: [10166]  PID: [5472] ReviewService : ServiceConnectionImpl.onServiceConnected(ComponentInfo{com.android.vending/com.google.android.finsky.inappreviewservice.InAppReviewService})
 I/PlayCore: UID: [10166]  PID: [5472] ReviewService : linkToDeath
 I/PlayCore: UID: [10166]  PID: [5472] OnRequestInstallCallback : onGetLaunchReviewFlowInfo
 I/PlayCore: UID: [10166]  PID: [5472] ReviewService : Unbind from service.
 I/Timeline: Timeline: Activity_launch_request time:57773147 intent:Intent { cmp=com.companyname.myapp/com.google.android.play.core.common.PlayCoreDialogWrapperActivity (has extras) }
 I/ActivityManager: START u0 {cmp=com.companyname.myapp/com.google.android.play.core.common.PlayCoreDialogWrapperActivity (has extras)} from uid 10166
E/ANDR-PERF-MPCTL: Invalid profile no. 0, total profiles 0 only
 D/ActivityTrigger: activityStartTrigger: Activity is Triggerred in full screen ApplicationInfo{e6f7b40 com.companyname.myapp}
 E/ActivityTrigger: activityStartTrigger: not whiteListedcom.companyname.myapp/com.google.android.play.core.common.PlayCoreDialogWrapperActivity/16
 D/CompatibilityInfo: mCompatibilityFlags - 0
 D/CompatibilityInfo: applicationDensity - 320
 D/CompatibilityInfo: applicationScale - 1.0
 D/ActivityTrigger: activityResumeTrigger: The activity in ApplicationInfo{e6f7b40 com.companyname.myapp} is now in focus and seems to be in full-screen mode
 E/ActivityTrigger: activityResumeTrigger: not whiteListedcom.companyname.myapp/com.google.android.play.core.common.PlayCoreDialogWrapperActivity/16
 D/ActivityTrigger: ActivityTrigger activityPauseTrigger 
 D/ActivityTrigger: activityResumeTrigger: The activity in ApplicationInfo{e6f7b40 com.companyname.myapp} is now in focus and seems to be in full-screen mode
 E/ActivityTrigger: activityResumeTrigger: not whiteListedcom.companyname.myapp/com.google.android.play.core.common.PlayCoreDialogWrapperActivity/16`

**Comment by NgocNamFNT** [comment]
No Popup display to rate in app android

**Comment by MinaSamir11** [comment]
update to the latest version and check your issue again.
and you must not see logs, which means you are trying to run dialog in debug mode and it's not working in debugging, you must upload it again to play store after integrating with the package. 

Read updated README at Android notes well.
https://github.com/MinaSamir11/react-native-in-app-review#-android-notes.
